cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java

```diff
@@ -29,6 +29,12 @@
 
     public static synchronized void main(String[] args) throws Exception
     {
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
```

Comment:
```suggestion
        if (Float.parseFloat(System.getProperty("java.class.version")) < 52F) 
```

Either compare Float to Float or Double to Double